README.md

```diff
@@ -153,7 +153,7 @@ Refer to the [LiteLLM Documentation](https://docs.litellm.ai/docs/providers) for
 ### Usage
 
 ```python
-from py_zerox import zerox
+from zerox import zerox
```

Comment:
Is this correct? With the package as `py-zerox` on the pip registry. Maybe this just pulls from the `.toml` name? First time I've published a py package so just not sure. 

https://pypi.org/project/py-zerox/

Reply:
@tylermaran, note the "py_zerox" folder it not a module, the actual module is the "zerox" folder in the py_zerox directory and that is what determines the import name. And different to all "py-zerox" is package name used to install with pip. 

Let me know what import name you prefer and I can make the required changes. I have "pyzerox" in mind, so the import will be - "from pyzerox import zerox".

Reply:
@tylermaran, yes XYZ should be unique, so as pypi name but you necessarily don't need them to be exactly same. Most important is XYZ should not clash/conflict with any other package like the "zerox" (not ours) package on pypi.

I'll update the import names.